cfbs/commands.py

```diff
@@ -207,7 +209,10 @@ def init_command(index=None, masterfiles=None, non_interactive=False) -> int:
 
     config["git"] = do_git
 
-    write_json(cfbs_filename(), config)
+    data = pretty(config, CFBS_DEFAULT_SORTING_RULES) + "\n"
```

Comment:
Q: Why not write it through the `cfbs_json` object like we do most other things?

Reply:
Feel free to take a stab at it, shouldn't be **very** hard.